packages/lit-dev-content/site/docs/templates/directives.md

```diff
@@ -11,33 +11,200 @@ versionLinks:
 Directives are functions that can extend Lit by customizing the way an expression renders.
 Lit includes a number of built-in directives to help with a variety of rendering needs:
 
-**Styling:**
-*   [`classMap`](#classmap) - sets a list of classes to an element based on an object
-*   [`styleMap`](#stylemap) - sets a list of style properties to an element based on an object
-
-**Rendering special values:**
-*   [`repeat`](#repeat) - renders values from an iterable into the DOM, with optional keying
-*   [`templateContent`](#templatecontent) - renders the content of a `<template>` element
-*   [`unsafeHTML`](#unsafehtml) - renders a string as HTML rather than text
-*   [`unsafeSVG`](#unsafesvg) - renders a string as SVG rather than text
-
-**Conditional rendering:**
-*   [`cache`](#cache) - caches rendered DOM when changing templates rather than discarding the DOM
-*   [`guard`](#guard) - only re-evaluates the template when one of its dependencies changes
-*   [`ifDefined`](#ifdefined) - sets an attribute if the value is defined and removes the attribute if undefined
-*   [`live`](#live) - sets an attribute or property if it differs from the live DOM value rather than the last-rendered value
-
-**Referencing the rendered DOM:**
-*   [`ref`](#ref) - gets a reference to an element rendered in the template
-
-**Asynchronous rendering:**
-*   [`until`](#until) - renders placeholder content until one or more promises resolve
-*   [`asyncAppend`](#asyncappend) - appends values from an `AsyncIterable` into the DOM as they are yielded
-*   [`asyncReplace`](#asyncreplace) - renders the latest value from an `AsyncIterable` into the DOM as it is yielded
+<table>
+  <tr><th>Directive</th><th>Summary</th></tr>
+  <tr><th colspan="2">Styling </th></tr>
+  <tr>
+  <td>
+
+  [`classMap`](#classmap)
+
+  </td>
+  <td>Sets a list of classes to an element based on an object </td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`styleMap`](#stylemap)
+
+  </td>
+  <td>Sets a list of style properties to an element based on an object</td>
+  </tr>
+
+  <tr><th colspan="2">Loops and Conditionals</th></tr>
+
+  <tr>
+  <td>
+
+  [`when`](#when)
+
+  </td>
+  <td>Renders one of two templates based on a condition</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`choose`](#choose)
+
+  </td>
+  <td>Renders one of many templates based on a key value</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`map`](#map)
+
+  </td>
+  <td>Transforms an iterable with a function</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`repeat`](#repeat)
+
+  </td>
+  <td>Renders values from an iterable into the DOM, with optional keying to enable data diffing and DOM stability</td>
+  </tr>
+
+  <tr>
+  <td>
+
+  [`join`](#join)
+
+  </td>
+  <td>Alternative values from an iterable with a joiner value</td>
```

Comment:
```suggestion
  <td>Alternate values from an iterable with a joiner value</td>
```

Reply:
Could also rephrase it more closely to how you did below, using the word "interleaved".